### ISSUE TITLE
Allow inserting VCF's manually (UID handling)

### ISSUE THREAD

**Issue by GSI** [scheibler/khard#106]
Sometimes I just insert a VCF file from some other source into the khard collection naming it manually like **Name_Surname.vcf**.

In this case they don't have an UID field.

So when I later move such a contact to a different address book (via `khard move`), the file will suddenly be named just **.vcf** (dot vcf).

Even if I first `khard modify` the contact, it will still not receive an UID.

Maybe khard should just fall back to maintaining the original file name?

Or maybe any **khard** command that does writes to or on the file should ensure that an UID is set?

**Comment by scheibler** [comment]
Yes, you are right. I've fixed it in the following way:

1. New contacts, created by khard, get a random uid during creation (file name is uid.vcf).

2. Modified contacts get a new random uid (if uid was missing) but only, if the contact was changed
   (file name does not change at all).

3. During copy/move a new random uid for the target contact is created, if the source contains no
   uid or the contact should be copied (file name for target is uid.vcf). So it's guaranteed, that the
   target contact has a unique uid and file name, even after copying.  I've dropped the idea to
   preserve the contacts file name at this point for better maintenance.

Conclusion:
1. It's not guaranteed, that file name and uid match
2. File name of contact is not preserved during copy/move

Does that work for you?
